.github/workflows/test_pr.yml

```diff
@@ -34,4 +34,36 @@ jobs:
         with:
           args: deploy -P roster-pr --only hosting:project-dashboard-pull-request
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
\ No newline at end of file
+          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+          
+# Only run the coverage once
+    name: get value percentage and print
+    run: 
```

Comment:
check the previous steps for correct syntax, indentation must be correct and something like

```
- name: Name of the step
  run: |
    use | for multiline commands 
    commands here
```
```
.github/workflows/test_pr.yml#L44
You have an error in your yaml syntax on line 44
```